views/show.ejs

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/public/style.css">
```

Comment:
Importing a CSS library makes styling much easier. I would recommend Bootstrap and Bulma.

Reply:
i second that